app/models/fee_scheme.rb

```diff
@@ -1,4 +1,18 @@
-class FeeScheme
+class FeeScheme < ActiveRecord::Base
+  validates :start_date, :number, :name, presence: true
+
+  has_many :offence_fee_schemes
+  has_many :offences, through: :offence_fee_schemes
+
+  scope :agfs, -> { where(name: 'AGFS') }
+  scope :lgfs, -> { where(name: 'LGFS') }
+  scope :current, lambda {
```

Comment:
**[not a blocker/nit pick]** This returns a _list_ of fee schemes that are currently active or in use, correct? Maybe it's just my own interpretation, but the first time I've looked at the method it suggested it would return **the** current **one**.

Reply:
perhaps `:active` instead but either way